_minutes/2025-02-13-wecg.md

```diff
@@ -0,0 +1,139 @@
+
+
+## Agenda: [discussion in #754](https://github.com/w3c/webextensions/issues/754), [github issues](https://github.com/w3c/webextensions/issues)
+
+The meeting will start at 3 minutes after the hour.
+
+See [issue 531](https://github.com/w3c/webextensions/issues/531) for an explanation of this agenda format.
+
+ * **Announcements** (2 minutes)
+   * Announcement: WECG meetup in Berlin, March 2025 ([Issue 759](https://github.com/w3c/webextensions/issues/759), [wiki page](https://github.com/w3c/webextensions/wiki/2025-Berlin-F2F-Coordination))
+ * **Triage** (15 minutes)
+   * [Issue 755](https://github.com/w3c/webextensions/issues/755): Proposal: about: matching
+   * [Issue 756](https://github.com/w3c/webextensions/issues/756): Proposal: Exclude Matching
+   * [Issue 762](https://github.com/w3c/webextensions/issues/762): DNR: Add excludedTopFrameDomains (and topFrameDomains)
+   * [Issue 763](https://github.com/w3c/webextensions/issues/763): Proposal: Add excludeTopFrameMatches and excludeTopFrameGlobs
+ * **Timely issues** (10 minutes)
+   * [PR 760](https://github.com/w3c/webextensions/pull/760): Specification for loading extensions in WebDriver Classic
+   * [issue 764](https://github.com/w3c/webextensions/issues/764): WebDriver: install command optional parameters
+ * **Check-in on existing issues** (20 minutes)
+   * [Issue 746](https://github.com/w3c/webextensions/issues/746): Proposal: Persistent Authentication
+
+
+## Attendees (sign yourself in)
+
+ 1. Rob Wu (Mozilla)
+ 2. Oliver Dunk (Google)
+ 3. Giorgio Maone (NoScript, Tor)
+ 4. Steven McLintock (1Password)
+ 5. Krzysztof Modras (Ghostery)
+ 6. Lena Cohen (Privacy Badger, EFF)
+ 7. Tomislav Jovanovic (Mozilla)
+ 8. Maxim Topciu (AdGuard)
+ 9. David Johnson (Apple)
+ 10. Kiara Rose (Apple)
+ 11. Mukul Purohit (Microsoft)
+ 12. Timothy Hatcher (Apple)
+ 13. Casey Garland (Capital One)
+
+
+## Meeting notes
+
+Announcement: WECG meetup in Berlin, March 2025 ([Issue 759](https://github.com/w3c/webextensions/issues/759), [wiki page](https://github.com/w3c/webextensions/wiki/2025-Berlin-F2F-Coordination))
+
+ * [tomislav] The in-person WECG meetup will take place from Tuesday 25 March until Friday 28 March, in the Mozilla office in Berlin. We expect about 10 - 20 attendees. If interested, sign yourself up in the issue.
+
+[Issue 755](https://github.com/w3c/webextensions/issues/755): Proposal: about: matching
+
+ * [timothy] about:-matching would be OK by me.
+ * [rob] about: in content scripts is already covered by match_origin_as_fallback; API-dependent solutions may work better. E.g. for context menus.
+ * [krzysztof] Do extensions have access to about:newtab at the moment?
+ * [rob] No.
+ * [krzysztof] Seems to have potential for privilege violation.
+ * [timothy] If anything, I would at most be supportive of about:blank and about:srcdoc.
+ * [oliver] Similarly. We don't even have about:newtab.
+ * [oliver] contextMenus API specifically could. (...)
```

Comment:
@oliverdunk Do you recall what was said here?

Reply:
I'm not certain, but here's a suggestion for something I said at one point at least.

```suggestion
 * [oliver] The contextMenus API use case specifically could be solved by a combination of `documentUrlPatterns` (to include the HTTP and HTTPS protocols, while excluding file:) and the addition of a `match_origin_as_fallback` like key that would also allow matching blank frames.
```